gooseai/generation/generation.pb.go

```diff
@@ -3126,104 +3195,107 @@ func file_generation_proto_rawDescGZIP() []byte {
+	3,  // 21: gooseai.TransformType.diffusion:type_name -> gooseai.DiffusionSampler
+	4,  // 22: gooseai.TransformType.upscaler:type_name -> gooseai.Upscaler
+	26, // 23: gooseai.ImageParameters.transform:type_name -> gooseai.TransformType
+	20, // 24: gooseai.ImageParameters.parameters:type_name -> gooseai.StepParameter
+	28, // 25: gooseai.ClassifierCategory.concepts:type_name -> gooseai.ClassifierConcept
+	7,  // 26: gooseai.ClassifierCategory.action:type_name -> gooseai.Action
+	8,  // 27: gooseai.ClassifierCategory.classifier_mode:type_name -> gooseai.ClassifierMode
+	29, // 28: gooseai.ClassifierParameters.categories:type_name -> gooseai.ClassifierCategory
+	29, // 29: gooseai.ClassifierParameters.exceeds:type_name -> gooseai.ClassifierCategory
+	7,  // 30: gooseai.ClassifierParameters.realized_action:type_name -> gooseai.Action
+	9,  // 31: gooseai.AssetParameters.action:type_name -> gooseai.AssetAction
+	10, // 32: gooseai.AssetParameters.use:type_name -> gooseai.AssetUse
+	32, // 33: gooseai.Answer.meta:type_name -> gooseai.AnswerMeta
+	14, // 34: gooseai.Answer.artifacts:type_name -> gooseai.Artifact
 	1,  // 35: gooseai.Request.requested_type:type_name -> gooseai.ArtifactType
-	15, // 36: gooseai.Request.prompt:type_name -> gooseai.Prompt
-	26, // 37: gooseai.Request.image:type_name -> gooseai.ImageParameters
-	29, // 38: gooseai.Request.classifier:type_name -> gooseai.ClassifierParameters
-	30, // 39: gooseai.Request.asset:type_name -> gooseai.AssetParameters
-	17, // 40: gooseai.Request.conditioner:type_name -> gooseai.ConditionerParameters
-	0,  // 41: gooseai.OnStatus.reason:type_name -> gooseai.FinishReason
-	10, // 42: gooseai.OnStatus.action:type_name -> gooseai.StageAction
-	33, // 43: gooseai.Stage.request:type_name -> gooseai.Request
-	34, // 44: gooseai.Stage.on_status:type_name -> gooseai.OnStatus
-	35, // 45: gooseai.ChainRequest.stage:type_name -> gooseai.Stage
-	33, // 46: gooseai.GenerationService.Generate:input_type -> gooseai.Request
-	36, // 47: gooseai.GenerationService.ChainGenerate:input_type -> gooseai.ChainRequest
-	32, // 48: gooseai.GenerationService.Generate:output_type -> gooseai.Answer
-	32, // 49: gooseai.GenerationService.ChainGenerate:output_type -> gooseai.Answer
-	48, // [48:50] is the sub-list for method output_type
-	46, // [46:48] is the sub-list for method input_type
-	46, // [46:46] is the sub-list for extension type_name
-	46, // [46:46] is the sub-list for extension extendee
-	0,  // [0:46] is the sub-list for field type_name
+	16, // 36: gooseai.Request.prompt:type_name -> gooseai.Prompt
+	27, // 37: gooseai.Request.image:type_name -> gooseai.ImageParameters
+	30, // 38: gooseai.Request.classifier:type_name -> gooseai.ClassifierParameters
+	31, // 39: gooseai.Request.asset:type_name -> gooseai.AssetParameters
+	18, // 40: gooseai.Request.conditioner:type_name -> gooseai.ConditionerParameters
+	2,  // 41: gooseai.Request.weight_method:type_name -> gooseai.WeightMethod
```

Comment:
It looks like the addition of 2: WeightMethod (i.e. displacing 2: DiffusionSampler instead of being inserted at the first available index at the high end of the number range) has shifted all the later numbers up. I may want to double-check that this doesn't have backwards compatibility implications.

Reply:
Jim helped to clarify that this is just an internal offset thing which doesn't affect backwards compatibility. So this should be fine.